src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -162,25 +259,28 @@ export const Checkout = ({
   const [emailUpdateLoading, setEmailUpdateLoading] = useState(false)
   const [ssnUpdateLoading, setSsnUpdateLoading] = useState(false)
   const [isShowingFailModal, setIsShowingFailModal] = useState(false)
+
+  const offerData = getOfferData(selectedQuoteBundleVariant.bundle)
+  const quoteIds = getQuoteIds(offerData)
```

Comment:
I would try to avoid this -- now we are doing the transformation in two places (OfferNew/index + here). I think we gain a lot from keeping that limited to a single place in the code. For example, the migration to the new API will be more complicated if we do this transformation in multiple places.

Reply:
I see that. It's kind of a complicated problem. Do you think this would be weird:

```
<Checkout
   offerData={offerData}
   bundleVariants={bundleVariants}
   selectedBundleVariant={selectedBundleVariant}
 />
```

Another alternativa would be renders `Checkout` content as children inside `OfferNew`. That might make `OfferNew` a huge thing.

Reply:
Yeah, complexity can usually only be moved around to different places. I don't have a clear idea what will be the most logical setup in this case.

I guess in general we should only pas the information necessary to `Checkout`. But I'm not sure what that would be.

Reply:
As we've discussed, this is a larger issue around the how we handle the data that is pretty much used everywhere is the page.